src/Storage/Device/S3.php

```diff
@@ -55,7 +85,7 @@ public function getPath($filename):string
      */
     public function upload($source, $path):bool
```

Comment:
missing types for $source and $path

Reply:
This is because the `Device` has as such, if i mention type here, I get an error

Reply:
You can fix this at the device level as well 👍🏻